modules/simple-payments/simple-payments.php

```diff
@@ -286,6 +287,134 @@ function allow_sync_post_meta( $post_meta ) {
 		) );
 	}
 
+	/**
+	 * Enable Simple payments custom meta values for access through the REST API.
+	 * Field’s value will be exposed on a .meta key in the endpoint response,
+	 * and WordPress will handle setting up the callbacks for reading and writing
+	 * to that meta key.
+	 *
+	 * @link https://developer.wordpress.org/rest-api/extending-the-rest-api/modifying-responses/
+	 */
+	public function register_meta_fields_in_rest_api() {
+		register_meta( 'post', 'spay_price', array(
+			'description'       => esc_html__( 'Simple payments; price.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_price' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'number',
+		) );
+
+		register_meta( 'post', 'spay_currency', array(
+			'description'       => esc_html__( 'Simple payments; currency code.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_currency' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_cta', array(
+			'description'       => esc_html__( 'Simple payments; text with "Buy" or other CTA', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'sanitize_text_field',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_multiple', array(
+			'description'       => esc_html__( 'Simple payments; allow multiple items', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'absint',
```

Comment:
I understand that this is a boolean, should we use [`rest_sanitize_boolean`](https://developer.wordpress.org/reference/functions/rest_sanitize_boolean/)?